app/controllers/onboarding_controller.rb

```diff
@@ -0,0 +1,19 @@
+class OnboardingController < ApplicationController
+  before_action :authenticate
+
+  def hood_and_houses
+    permitted_params = params.permit(entries: Hood::Onboarder::RequiredFields)
```

Comment:
Example request that:

- Creates two neighborhoods ("Schitt's Creek" and "Star City").
- Creates two houses in "Star City".

```json
{
  "entries": [
    {
      "hood_name": "Schitt's Creek",
      "hood_zip_code": "60601",
      "house_street_address": "",
      "house_city": "",
      "house_state": "",
      "house_zip_code": "",
      "house_bedrooms": "",
      "house_bathrooms": "",
      "house_square_feet": ""
    },
    {
      "hood_name": "Star City",
      "hood_zip_code": "60622",
      "house_street_address": "123 Fake St",
      "house_city": "Chicago",
      "house_state": "IL",
      "house_zip_code": "60622",
      "house_bedrooms": "3",
      "house_bathrooms": "1",
      "house_square_feet": "1200"
    },
    {
      "hood_name": "Star City",
      "hood_zip_code": "60622",
      "house_street_address": "125 Fake St",
      "house_city": "Chicago",
      "house_state": "IL",
      "house_zip_code": "60622",
      "house_bedrooms": "2",
      "house_bathrooms": "2",
      "house_square_feet": "1000"
    }
  ]
}
```